code/modules/power/lighting.dm

```diff
@@ -42,6 +42,7 @@
 	icon_state = "tube-construct-stage1"
 	anchored = TRUE
 	layer = WALL_OBJ_LAYER
+	plane = LIGHTING_PLANE
```

Comment:
This is gonna make light fixture construction sprites add onto all lower planes multiplicatively, and make them inherently mouse transparent